src/ws/test-remotectlcertificate.c

```diff
@@ -224,7 +234,8 @@ test_refresh_expired (TestCase *test,
   g_autofree gchar *oldpath = g_build_filename (test->cert_dir, "alice-expired.cert", NULL);
   g_autofree gchar *selfsigned_path = g_build_filename (test->cert_dir, "0-self-signed.cert", NULL);
   g_autoptr(GTlsCertificate) certificate = NULL;
-  char *argv[] = { "certificate", "--user", (gchar *) g_get_user_name (), "--ensure", NULL };
+  char *argv[] = { "certificate", "--ensure", "--user", (gchar *) g_get_user_name (),
+                   test->group ? "--group" : NULL, test->group, NULL };
```

Comment:
...this made me raise an eyebrow... "gross, but okay, kinda creative... and it's just a test..."

Reply:
Dropped. As I wrote above, I made the unit test stricter. Nobody has complained in years that they want to build cockpit in an environment where the current user does not have a group name, so we can just as well make it official. :grin: